<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSelEsquema</class>
 <widget class="QDialog" name="frmSelEsquema">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seleccione d√≥nde buscar</string>
  </property>
  <widget class="QListWidget" name="lwTipoTrabajo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lwSrc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>271</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccione el tipo de trabajo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccione el sistema de coordenadas:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bttAceptar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
